internal/compiler-bridge/src/main/scala/xsbt/Command.scala

```diff
@@ -7,21 +7,31 @@
 
 package xsbt
 
-import scala.tools.nsc.{ CompilerCommand, Settings }
+import scala.tools.nsc.{CompilerCommand, Settings}
```

Comment:
This (and other examples) don't convince me we're really using the same configuration here..